sagemaker-pipelines/tabular/tuning-step/tuning_step_example.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>can we please also add a section to clean up the resources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>